tsconfig.json

```diff
@@ -1,8 +1,9 @@
 {
-  "include": ["src"],
+  "include": ["src/**/*"],
+  "exclude": ["src/**/*.test.ts"],
   "compilerOptions": {
-    "target": "es2018",
-    "module": "esnext",
+    "target": "es2020",
+    "module": "CommonJS",
```

Comment:
why are we changing from esnext to commonJS?

Reply:
@just-toby When we dropped the deprecated build tools library, we had to choose between commonjs and es modules. As far as I could see, commonjs compiles are usable with both commonjs and es modules. Whereas the other way round is only compatible with es modules (not relevant for typescript but still relevant for plain JS).

If you have reasons to switch to es modules only and are ok with support for older JS projects being dropped I can go ahead.

Reply:
I think forcing people to use es modules is sensible.

Reply:
especially considering we were already using esm, it would risk breaking other things (eg tree-shaking) if we switched it now.